public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -100,208 +97,213 @@ code-example(language="sh" class="code-shell").
-    At the moment, it is still referring to the old `hero` property.
-    Let’s fix the template to bind to the new `selectedHero` property.
+    At the moment, the template still refers to the old `hero` property.
+    To change the template to bind to the new `selectedHero` property, update the code as follows:
 
-  +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'selectedHero-details', 'app.component.ts (template excerpt)')
+  +makeExample('toh-2/ts/app/app.component.1.html', 'selectedHero-details', 'app.component.ts (template excerpt)')(format='.')
   :marked
     ### Hide the empty detail with ngIf
 
-    When our app loads we see a list of heroes, but a hero is not selected.
+    When the app loads, a list of heroes displays, but no hero name is selected.
     The `selectedHero` is `undefined`.
-    That’s why we'll see the following error in the browser’s console:
+    Thus, the following error appears in the browser's console:
 
   code-example(format="nocode").
     EXCEPTION: TypeError: Cannot read property 'name' of undefined in [null]
 
   :marked
-    Remember that we are displaying `selectedHero.name` in the template.
-    This name property does not exist because `selectedHero` itself is undefined.
+    Although `selectedHero.name` is displayed in the template,
+    this name property doesn't exist because `selectedHero` itself is undefined.
+    So for now, keep the hero detail out of the DOM until there is a selected hero.
 
-    We'll address this problem by keeping the hero detail out of the DOM until there is a selected hero.
+    Wrap the HTML hero detail content of your template with a `<div>`.
+    Then add the `ngIf` built-in directive and set it to the `selectedHero` property of the component.
 
-    We wrap the HTML hero detail content of our template with a `<div>`.
-    Then we add the `ngIf` built-in directive and set it to the `selectedHero` property of our component.
+  +makeExample('toh-2/ts/app/app.component.1.html', 'ng-if', 'app.component.ts (ngIf)')(format='.')
 
-  +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'ng-if', 'app.component.ts (ngIf)')
+  :marked
+    The list of names displays again in the browser.
 
   .alert.is-critical
     :marked
       Remember that the leading asterisk (`*`) in front of `ngIf` is
       a critical part of this syntax.
+  //  CF: is a "critical" type note appropriate here? Could I merge this sentence with
+    the next section?
   :marked
     When there is no `selectedHero`, the `ngIf` directive removes the hero detail HTML from the DOM.
-    There will be no hero detail elements and no bindings to worry about.
+    There are no hero detail elements or bindings to worry about.
 
     When the user picks a hero, `selectedHero` becomes "truthy" and
     `ngIf` puts the hero detail content into the DOM and evaluates the nested bindings.
+  //  CF: Is "truthy" jargon? How about: "`selectedHero` is set to true ..."
+
   .l-sub-section
```

Comment:
This sub section is unnecessary in the tutorial. Who cares what they're called?

Reduce it to
```
      Read more about `ngIf` and `ngFor` in the
      [Structural Directives](../guide/structural-directives.html) page and the
      [_Built-in directives_](../guide/template-syntax.html#directives) section of the
      [Template Syntax](../guide/template-syntax.html) page.
```